db/migrate/20170515161016_reindex_account_info.rb

```diff
@@ -0,0 +1,27 @@
+class ReindexAccountInfo < ActiveRecord::Migration
+  def change
+    url = "http://#{CONFIG['elasticsearch_host']}:#{CONFIG['elasticsearch_port']}"
+    client = Elasticsearch::Client.new url: url
+    options = {
+      index: CONFIG['elasticsearch_index'].blank? ? [Rails.application.engine_name, Rails.env, 'annotations'].join('_') : CONFIG['elasticsearch_index'],
```

Comment:
There are already 2 places in the code that use this same line to get the ES index name. Please make a function in `CheckElasticSearchModel` that returns the index name and call it instead of this line.